src/App.jsx

```diff
@@ -14,70 +17,78 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [value, setValue] = useState('Jam');
```

Comment:
this state name is generic
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```